docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -104,65 +103,60 @@ Next, set API permissions for Rancher within Azure.
 
-    :::note
+![Open Required Permissions](/img/select-req-permissions.png)
+
+:::note
+Rancher does not validate the permissions you grant to the app in Azure. Technically, you just need to grant a set of permissions that are sufficient for Rancher to do work with AD.
 
-    You must be signed in as an Azure administrator to successfully save your permission settings.
+For example, another set of acceptable permissions includes `Group.Read.All` and `User.Read.All` (of type Application). But the sole `Directory.Read.All` (also of type Application)
+is more narrow than the other two, hence it is recommended. If you configure a custom set of permissions instead, be aware that Rancher may not be able to perform some of the actions
+within Azure AD, if the permissions are insufficient, so you may encounter errors.
 
-    :::
+:::
 
 #### 4. Copy Azure Application Data
 
 As your final step in Azure, copy the data that you'll use to configure Rancher for Azure AD authentication and paste it into an empty text file.
 
+![Application ID](/img/app-configuration.png)
+
 1. Obtain your Rancher **Tenant ID**.
 
     1. Use search to open **App registrations**.
 
-    ![Open App Registrations](/img/search-app-registrations.png)
-
     1. Find the entry you created for Rancher.
 
-    1. Copy the **Directory ID** and paste it into your [text file](#tip).
-
-    ![Tenant ID](/img/tenant-id.png)
-
-    - You'll paste this value into Rancher as your **Tenant ID**.
+    1. Copy the **Directory ID** and paste it into your text file. You'll paste this value into Rancher as your **Tenant ID**.
 
 1. Obtain your Rancher **Application (Client) ID**.
 
-    2.1. Use search to open **App registrations** (if not already there).
-
-    2.2. In **Overview**, find the entry you created for Rancher.
+    1. Use search to open **App registrations** (if not already there).
 
-    2.3. Copy the **Application (Client) ID** and paste it to your [text file](#tip).
+    1. In **Overview**, find the entry you created for Rancher.
 
-    ![Application ID](/img/application-client-id.png)
+    1. Copy the **Application (Client) ID** and paste it to your text file.
 
-1. Your endpoint options will typically be [Standard](#global) and [China](#china). With these options, you need only enter the **Tenant ID**, **Application ID**, and **Application Secret** (Rancher will take care of the rest).
+1. Your endpoint options will typically be either [Standard](#global) or [China](#china). With these options, you only enter the **Tenant ID**, **Application ID**, and **Application Secret**.
 
 ![Standard Endpoint Options](/img/tenant-application-id-secret.png)
 
 **For Custom Endpoints:**
 
-**Warning:** Custom Endpoints are not supported nor fully tested by Rancher.
+**Warning:** Custom Endpoints are not tested and thus not fully supported by Rancher.
```

Comment:
Nit: I would rephrase this  as `Custom Endpoints are not tested or fully supported by Rancher`. The way it is currently written, it implies that if they were tested then they would be supported, which I don't think we want to imply.